sklearn/svm/classes.py

```diff
@@ -906,6 +906,19 @@ class SVR(BaseLibSVM, RegressorMixin):
         `coef_` is readonly property derived from `dual_coef_` and
         `support_vectors_`.
 
+    class_weight_ : array, shape = [n_class]
```

Comment:
This one doesn't really make sense for regression. There is no ``n_class``. Can you please double check if it actually exists and what it is?

Reply:
class_weight will be np.empty(0) by default as a parameter in libsvm.pyx  -- which is why it exists? -- `class_weight : array, dtype float64, shape (n_classes,), optional`